README.rst

```diff
@@ -65,6 +65,13 @@ Features
 
 -----
 
+=============
+Prerequisites
+=============
+
+* Python >= 3.4
+* pip >= 8.0
```

Comment:
Error on my part. Github and git email addresses were different. It is fixed now and other commits won't have this problem.